packages/opal-server/opal_server/policy/webhook/api.py

```diff
@@ -71,10 +71,11 @@ def get_webhook_router(
         dependencies=route_dependencies,
     )
     async def trigger_webhook(request: Request, git_changes: GitChanges = git_changes):
-        # look at values extracted from request
-        urls = git_changes.urls
-        branch = git_changes.branch
-        names = git_changes.names
+        if source_type != PolicySourceTypes.Api:
```

Comment:
you could have just moved these into the if at 81 (i think)

Reply:
to make sure it wont skip for any other source types in future.

Reply:
I agree with @michaelherrmann-nutrien , Those are only relevant for Git source type (even if other types would be introduced in the future).

Just move those inside the if :)
And then I'll approve...

Reply:
Updated the PR as requested 👍